FrameworkSystem/scripts/dirac-populate-component-db.py

```diff
@@ -68,6 +71,10 @@ def setExcludedHosts( value ):
 if not resultInfo[ 'OK' ]:
   gLogger.error( resultInfo[ 'Message' ] )
   DIRACexit( -1 )
+resultMySQL = client.getMySQLStatus()
+if not resultMySQL[ 'OK' ]:
+  gLogger.error( resultMySQL[ 'Message' ] )
+  DIRACexit( -1 )
```

Comment:
If MySQL is not installed on the machine, it is not necessarily an issue. Or am I missing something?